Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -649,9 +649,6 @@ const ScrollView = React.createClass({
         {...contentSizeChangeProps}
         ref={this._setInnerViewRef}
         style={contentContainerStyle}
-        removeClippedSubviews={
-          hasStickyHeaders && Platform.OS === 'android' ? false : this.props.removeClippedSubviews
-        }
```

Comment:
stickySectionHeader on `ListView` is not work as expected after this change on both master branch and 0.45.0-rc.0, while it's OK on `SectionList`
![image](https://cloud.githubusercontent.com/assets/2595058/26117863/7fb2d31e-3a9a-11e7-8082-ca8ee17f2026.png)
You can see the sticked sectionHeader is overlapped by the others, works perfectly after reverted